views/forgot.go

```diff
@@ -51,10 +56,9 @@ func (v *Views) ForgotFunc(w http.ResponseWriter, r *http.Request) {
 		// Get user1 and check if it exists
 		user1, err := v.user.GetUser(r.Context(), u)
 		if err != nil {
-			// User doesn't exist, we'll pretend they've got an email
+			// UserStripped doesn't exist, we'll pretend they've got an email
```

Comment:
This a renaming mistake or intentional?

Reply:
That may have been a result of a find and replace, has been fixed

Reply:
Still appears wrong in the latest commit, unless you mean fixed in a later PR?

Reply:
Fixed